### PR TITLE
Fixes ETH to DAI error

### LINKED ISSUE #477: WETH to DAI swaps fails
Cannot swap on mainnet nor on rinkeby.

### PR DESCRIPTION
**Pull request by robbestad**
Closes #477

### CODE REVIEW

File: src/components/TransactionsTab/TransactionsTab.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { useCallback, useEffect, useRef, useMemo, useState } from "react";
 import { useTranslation } from "react-i18next";
 
 import { TokenInfo } from "@airswap/types";
-import { getEtherscanWalletURL } from "@airswap/utils";
+import { getEtherscanURL } from "@airswap/utils";
```
Comment (piersss) [inline src/components/TransactionsTab/TransactionsTab.tsx:5]: `getEtherscanWalletURL` is for getting wallet url.
`getEtherscanURL` is for getting a transaction url.

File: src/components/SwapWidget/SwapWidget.tsx [hunk 2]
```diff
@@ -332,7 +332,11 @@ const SwapWidget: FC<SwapWidgetPropsType> = ({
   };
 
   const handleSetToken = (type: TokenSelectModalTypes, value: string) => {
-    const baseRoute = `${appRouteParams.justifiedBaseUrl}/${AppRoutes.swap}`;
+    const baseUrl =
+      appRouteParams.justifiedBaseUrl !== "/"
+        ? appRouteParams.justifiedBaseUrl
+        : "";
+    const baseRoute = `${baseUrl}/${AppRoutes.swap}`;
```
Comment (piersss) [inline src/components/SwapWidget/SwapWidget.tsx:339]: Something wrong with appRouteParams?

File: src/components/SwapWidget/SwapWidget.tsx [hunk 3]
```diff
@@ -348,6 +352,7 @@ const SwapWidget: FC<SwapWidgetPropsType> = ({
       dispatch(setUserTokens({ tokenFrom, tokenTo }));
     }
 
+    console.log(`${baseRoute}/${tokenFrom}/${tokenTo}`);
```
Comment (piersss) [inline src/components/SwapWidget/SwapWidget.tsx:355]: console.log

**Review by piersss — CHANGES_REQUESTED** [review]

**Comment by piersss** [comment]
> Closes #477
> 
> Reverts @airswap/utils. The method `createLightOrder` used in 0.4.10 does not exist in the latest @airswap/utils package and seemingly has no equivalent method that replaces it

Oh that sucks

File: src/components/SwapWidget/SwapWidget.tsx (hunk 2 above)
Comment (robbestad) [reply src/components/SwapWidget/SwapWidget.tsx:339]: yes, you get // and an error

File: src/components/SwapWidget/SwapWidget.tsx (hunk 3 above)
Comment (robbestad) [reply src/components/SwapWidget/SwapWidget.tsx:355]: Ok, fixing!

File: src/components/TransactionsTab/TransactionsTab.tsx (hunk 1 above)
Comment (robbestad) [reply src/components/TransactionsTab/TransactionsTab.tsx:5]: but you're right, I can't just swap `getEtherscanWalletURL ` for `getEtherscanURL`...

File: src/components/SwapWidget/SwapWidget.tsx (hunk 2 above)
Comment (piersss) [reply src/components/SwapWidget/SwapWidget.tsx:339]: Ah I see, it's broken.

**Comment by dmosites** [comment]
> > Closes #477
> > Reverts @airswap/utils. The method `createLightOrder` used in 0.4.10 does not exist in the latest @airswap/utils package and seemingly has no equivalent method that replaces it
> 
> Oh that sucks

Is `createOrder` not available in the latest package?
https://github.com/airswap/airswap-protocols/blob/main/tools/utils/index.ts#L29

File: src/components/SwapWidget/SwapWidget.tsx (hunk 2 above)
Comment (piersss) [reply src/components/SwapWidget/SwapWidget.tsx:339]: I fixed appRouteParams. Pushed as commit.

Comment (robbestad) [reply src/components/SwapWidget/SwapWidget.tsx]: great, I removed this piece of code now

**Comment by robbestad** [comment]
> `createLightOrder`

yes it is, but not `createLightSignature`

**Comment by robbestad** [comment]
> 

yes it is, but not createLightSignature

**Comment by robbestad** [comment]
> > Closes #477
> > Reverts @airswap/utils. The method `createLightOrder` used in 0.4.10 does not exist in the latest @airswap/utils package and seemingly has no equivalent method that replaces it
> 
> Oh that sucks

I didn't look hard enough.... :)

**Review by piersss — COMMENTED** [review]
ETH to DAI works. But DAI to ETH crashes still on testnet.

**Comment by piersss** [comment]
> > ETH to DAI works. But DAI to ETH crashes still on testnet.
> 
> I know but there ain't much I can do about that. It works on mainnet.

Ok will approve then.

**Review by piersss — APPROVED** [review]

**Comment by dmosites** [comment]
Small conflict remaining if you could take a look please @piersss

**Pull request merged**